### DISCUSSION TITLE
Concat conditions?

### DISCUSSION THREAD

**Discussion by sdk401** [cubiq/prompt_injection#3] [General]
Node gives an error when trying to pass concat conditions to any input. Is this a bug or a feature? Tried to battle the bleeding with this.

**Comment by cubiq** [comment]
we are working on the cross attention, I'm not sure we can do that kind of conditioning but I'll check. to do concat you can basically use multiple injections

**Reply by sdk401** [reply]
And how do i use multiple injections? Do I somehow combine the model outputs of multiple Injection nodes?

**Reply by cubiq** [reply]
just daisy chain the injection nodes
